ClusterClient.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Credis_Client (a fork of Redisent)
+ *
+ * Most commands are compatible with phpredis library:
+ *   - use "pipeline()" to start a pipeline of commands instead of multi(Redis::PIPELINE)
+ *   - any arrays passed as arguments will be flattened automatically
+ *   - setOption and getOption are not supported in standalone mode
+ *   - order of arguments follows redis-cli instead of phpredis where they differ (lrem)
+ *
+ * - Uses phpredis library if extension is installed for better performance.
+ * - Establishes connection lazily.
+ * - Supports tcp and unix sockets.
+ * - Reconnects automatically unless a watch or transaction is in progress.
+ * - Can set automatic retry connection attempts for iffy Redis connections.
+ *
+ * @author Colin Mollenhour <colin@mollenhour.com>
+ * @copyright 2011 Colin Mollenhour <colin@mollenhour.com>
+ * @license http://www.opensource.org/licenses/mit-license.php The MIT License
+ * @package Credis_Client
+ */
+
+/**
+ * Credis_ClusterClient, subclass to Credis_Client that uses RedisCluster
+ */
+class Credis_ClusterClient extends Credis_Client
+{
+    /**
+     * Name of the cluster as configured in redis.ini
+     * @var string|null
+     */
+    protected ?string $clusterName;
+
+    /**
+     * Hosts & ports of the cluster
+     * Eg: ['redis-node-1:6379', 'redis-node-2:6379', 'redis-node-3:6379', 'redis-node-4:6379']
+     * @var array|null
+     */
+    protected ?array $clusterSeeds;
+
+    /**
+     * Enable persistent connections
+     * @var bool
+     */
+    protected bool $persistentBool;
+
+
+    /**
+     * Creates a connection to the Redis server on host {@link $host} and port {@link $port}.
```

Comment:
Oops, I forgot to update the constructor description as this one is just copied from Credis_Client

Reply:
resolved in 1f467762a1f0bb1375fa391f0e793e98eb033f1b